src/System.Diagnostics.Contracts/tests/packages.config

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
```

Comment:
You maybe correct I've not looked to closely I was just expecting the elimination of packages.config.